jiant/models.py

```diff
@@ -286,9 +286,12 @@ def build_model(args, vocab, pretrained_embs, tasks):
     d_task_input = d_sent_output + (args.skip_embs * d_emb)
 
     # Build model and classifiers
-    model = MultiTaskModel(args, sent_encoder, vocab)
+    model = MultiTaskModel(args, sent_encoder, vocab, cuda_devices, use_cuda)
     build_task_modules(args, tasks, model, d_task_input, d_emb, embedder, vocab)
-    model = model.cuda() if args.cuda >= 0 else model
+    model = model.cuda() if use_cuda else model
+    if use_cuda:
+        model = nn.DataParallel(model, device_ids=cuda_devices).cuda()
```

Comment:
so if we use GPU, we always use DataParallel? Does that incur a big cost in the single GPU case?

Reply:
I _think_ for one DataParallel, it doesn't do anything (because there's only one GPU). I'll change this though.